commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPAttachmentFileEntryLocalServiceImpl.java

```diff
@@ -72,13 +77,15 @@ public CPAttachmentFileEntry addCPAttachmentFileEntry(
 			int expirationDateDay, int expirationDateYear,
 			int expirationDateHour, int expirationDateMinute,
 			boolean neverExpire, Map<Locale, String> titleMap, String json,
-			double priority, int type, ServiceContext serviceContext)
+			double priority, int type, long userId,
```

Comment:
Ideally the `userId` param should be sorted earlier, my preference would be the first param I think.